nat-lab/tests/test_events_link_state.py

```diff
@@ -117,6 +118,7 @@ async def test_event_link_state_peers_idle_all_time(
 
 
 @pytest.mark.asyncio
+@pytest.mark.xfail(reason="test flaky - JIRA issue: LLT-4695")
```

Comment:
I don't see this test in latest report: https://bucket.digitalarsenal.net/low-level-hacks/automation/gitlab/ci-helper-scripts/-/jobs/10767100

Reply:
Added in the description and in the issue. Thank you